common/types.go

```diff
@@ -542,3 +546,7 @@ func (l Location) ContainsAddress(a Address) bool {
 	// Ranges are fully inclusive
 	return prefix >= prefixRange.lo && prefix <= prefixRange.hi
 }
+
+func IsAddressInContext(address Address) bool {
```

Comment:
Since this only exists to facilitate `IsInContext()`, but `IsInContext()` is being removed, this should be removed too

Reply:
Removed